cadf/meta/logrotate.yml

```diff
@@ -0,0 +1,19 @@
+{%- from "cadf/map.jinja" import config with context %}
+{%- from "cadf/map.jinja" import dispatcher with context %}
+{%- from "cadf/map.jinja" import listener with context %}
+
+{%- if dispatcher.enabled or listener.enabled  %}
+job:
+  cadf:
+    - files: {{ config.log_dir }}/*.log
```

Comment:
can we do a change, backward compatible, and use following pillar structure? 
```
config:
  log:
    dir: /xyz
    logrotate:
       rotate: 7
       minsize: 50M
       maxsize: 100M
      ```

with these defaults in map.jinja

Reply:
I changed my mind, If you define log_dir fist time, then this way
 ```
config:
  log:
    dir: /xyz
```

This is not required for the merge, so keep the for the other day:
```
logrotate:
       rotate: 7
       minsize: 50M
       maxsize: 100M
```